Client/Frontend/Browser/Tab Management/Tab.swift

```diff
@@ -583,23 +583,34 @@ class Tab: NSObject {
         webView?.stopLoading()
     }
 
-    func reload() {
-        // If the current page is an error page, and the reload button is tapped, load the original URL
-        if let url = webView?.url, let internalUrl = InternalURL(url), let page = internalUrl.originalURLFromErrorPage {
-            webView?.replaceLocation(with: page)
-            return
-        }
-
-        if let _ = webView?.reloadFromOrigin() {
-            print("reloaded zombified tab from origin")
-            return
-        }
-
-        if let webView = self.webView {
-            print("restoring webView from scratch")
-            restore(webView)
-        }
-    }
+	func reload(bypassCache: Bool = false) {
```

Comment:
Please review indentation of your branch, we indent with 4 spaces in this project.

Reply:
Indentation was set to 4 spaces. Please advise on further issues.

Reply:
It's not set at 4 spaces, it's currently tab spacing. You should be able to adjust through Xcode settings and use CTRL+I shortcut command to use the proper indentation. All the code changed in this PR has tab spacing, so before I continue reviewing this need to be adjusted. 

Those are the settings I am referring to, please make sure you have those set properly to work on this project.
![Screen Shot 2022-10-26 at 5 29 47 PM](https://user-images.githubusercontent.com/11338480/198142206-a975e890-c718-4944-995a-d9dfd9f391e2.png)
![Screen Shot 2022-10-26 at 5 30 36 PM](https://user-images.githubusercontent.com/11338480/198142241-661ab4c4-9e8b-4569-b586-19730063e606.png)


As an example, pulling the branch and formatting locally I get the following diffs which shouldn't be the case.
![Screen Shot 2022-10-26 at 5 21 29 PM](https://user-images.githubusercontent.com/11338480/198140736-d2df71e3-ce2e-4e0e-9f42-bd58bba6de35.png)

Reply:
@lmarceau I changed to spaces for the specific files I worked on and reindented everything. Hopefully it will now be okay.

Reply:
It's better already! Just one more thing is that we setup the project so there's no [trailing whitespaces](https://realm.github.io/SwiftLint/trailing_whitespace.html). For that you need the following option: 
![Screen Shot 2022-10-27 at 9 53 55 AM](https://user-images.githubusercontent.com/11338480/198304003-2d5e4523-5594-47f5-a02b-9fb65b9b87e1.png)

As we see in Github there's the following diff in each file. 
![Screen Shot 2022-10-27 at 9 51 58 AM](https://user-images.githubusercontent.com/11338480/198304182-f9a78c52-309d-4030-be37-ba8c27e728b5.png)

You can then re-use the Ctrl+i shortcut in Xcode, or if you have Swiftlint on your machine you can run `swiftlint autocorrect` to fix all occurrences of this issue.

Reply:
@lmarceau Apologies for the back and forth... Seems my editor is not configured properly for this project yet. I have committed the changes to remove the trailing whitespace. Hopefully this is all sorted now? Although I see that by fixing the whitespacing, it formatted the switch case based on my default preferences which is to indent the "case". Is this a problem?